test/e2e/framework.go

```diff
@@ -1527,6 +1527,21 @@ func (data *TestData) createNginxPodOnNode(name string, ns string, nodeName stri
 	}).WithHostNetwork(hostNetwork).Create(data)
 }
 
+func (data *TestData) createUDPServerPod(name string, ns string, portNum int32, serverNode string) error {
```

Comment:
I would recommend not defining this helper function if it's only used once

Reply:
moved out of framework.